src/tests/ftest/soak/smoke.yaml

```diff
@@ -166,6 +168,30 @@ fio_smoke:
         mount_dir: "/tmp/daos_dfuse/fio/"
 daos_racer:
     runtime: 120
+vpic_smoke:
+    job_timeout: 10
+    nodesperjob:
+        - 1
+    taskspernode:
+        - 1
+    cmdline: "/home/mjean/vpic-install/bin/harris.Linux"
```

Comment:
We can't have paths to files that are going to only exist in a single user's configuration in our test framework.  Clearly this will only work for very isolated use-cases.

Reply:
soak will only run in CI to commit the code.    The jobs you highlight will not be run in CI because the jobs are commented out.   All job executables that run in CI will be installed from rpms (ior, mdtest, fio etc... )  but we are not creating rpms for the additional benchmarks that I am adding; we will use a local build and if the build location changes then the yaml is updated before it is run.